src/components/reusable-ui/Logo.jsx

```diff
@@ -1,20 +1,18 @@
-import React from 'react'
-import styled from 'styled-components';
-import { theme } from '../../theme';
+import styled from "styled-components";
+import { theme } from "../../theme";
 
-export default function Logo() {
+export default function Logo({ onClick }) {
   return (
-    <LogoStyled>
-        <h1>Crazee</h1> 
-        <img src="/images/logo-orange.png" alt="Logo Crazee Burger" />
-        <h1>Burger</h1>
+    <LogoStyled className="logo" onClick={onClick}>
```

Comment:
```suggestion
    <LogoStyled className={className} onClick={onClick}>
```
pour que le Logo soit réutilisable même au niveau du style. Sinon, tu vas forcer les autres dev à entrer à l'intérieur du composant et voir quel est le nom de la class pour changer le style de Logo.

Reply:
Yep je m'y colle